app/javascript/components/toggle-display.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class="toggle-display" :class="stateClass" @click.self="onClick" >
```

Comment:
Attribute "bind" should be on a new line  vue/max-attributes-per-line<br>Attribute "on" should be on a new line    vue/max-attributes-per-line